seqr/views/apis/saved_variant_api.py

```diff
@@ -92,14 +93,39 @@ def create_saved_variant_handler(request):
 @login_required(login_url=API_LOGIN_REQUIRED_URL)
 @csrf_exempt
 def create_variant_note_handler(request, variant_guid):
+    save_as_gene_note = json.loads(request.body).get('saveAsGeneNote')
```

Comment:
you should probably do `request_json = json.loads(request.body)` once at the begining fo this function, rather than re-parsing the json 3 times